Languages/IronPython/IronPythonTest/EngineTest.cs

```diff
@@ -1031,7 +1031,7 @@ class EmptyNC(object): pass
         public void ScenarioDlrInterop() {
             string actionOfT = typeof(Action<>).FullName.Split('`')[0];
 
-#if !SILVERLIGHT
+#if FEATURE_WINDOWS
```

Comment:
I thought Mono supported `System.Windows.Forms`. Perhaps we should create a `FEATURE_WINFORMS` instead?

In any case, only a small part of this test actually requires winforms. I will see if I can refactor it (unless you want to do it).